### PR TITLE
add real patch to series

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Since I couldn't find anything in our js using this API path I think we should be fine merging this.

### CODE REVIEW

File: medusa/server/api/v2/series.py [hunk 1]
```diff
@@ -96,18 +102,28 @@ def patch(self, series_slug, path_param=None):
...
-                elif value is False:
-                    series.unpause()
-                else:
-                    return self._bad_request('Invalid request body: pause')
-                done[key] = value
-
-        return self._ok(done)
+        accepted = {}
+        ignored = {}
+        patches = {
```
Comment (OmgImAlexis) [inline medusa/server/api/v2/series.py]: @ratoaq2 I know config uses self.patches but we need access to the series object which isn't available in the class scope.

Comment (p0psicles) [reply medusa/server/api/v2/series.py]: Ok if i'm understanding this correct, then what your doing here is getting the series object, and changing an attribute if it needs to be patched.

But the config uses this command, to store the new `in-mem` values into the db: `app.instance.save_config()`
You don't have something similar for the series object? Or i'm missing something. So these changes will not be persistent.

Comment (OmgImAlexis) [reply medusa/server/api/v2/series.py]: The patches is a list of patchable fields, this is literally copied and pasted from the other apiv2 end points for patching.

Comment (OmgImAlexis) [reply medusa/server/api/v2/series.py]: The only difference is config uses `self.patches` where as this uses a local variable called patches.

**Comment by p0psicles** [comment]
I need to reverse engineer the apiv2 to figure out how to use the patch method.
Don't we have this documented somewhere with some real examples?

**Comment by p0psicles** [comment]
Some notes.
For passing booleans, the api only accepts passing it as a `"scene": true` and not `"scene": "true"`.
Do we want to limit it to that?

**Comment by p0psicles** [comment]
@medariox you know why it's still hanging on code beat?

**Comment by medariox** [comment]
@p0psicles Try to rebase.

**Comment by OmgImAlexis** [comment]
@p0psicles 
> For passing booleans, the api only accepts passing it as a "scene": true and not "scene": "true".
Do we want to limit it to that?

We do this as the API accepts JSON so the booleans should be booleans not strings. I'd rather keep it that way if possible.

**Pull request merged**